hpaction/build_hpactionvars.py

```diff
@@ -64,10 +66,103 @@ def create_complaint(area: str, description: str) -> hp.TenantComplaints:
     )
 
 
+def nycdb_addr_to_hp_state(address: nycdb.models.Address) -> hp.LandlordAddressStateMC:
+    # This is kind of yucky, because NYCDB/HPD provides state information as
+    # all-caps two-letter strings (e.g. "NY") while the HotDocs endpoint wants
+    # them fully spelled out (e.g. "New York"). However, in practice it doesn't seem
+    # to mind if we give it the two-letter strings, so we'll just do that.
+    class OneOffEnum(Enum):
+        VALUE = address.state
+
+    return OneOffEnum.VALUE  # type: ignore
+
+
+def fill_landlord_info_from_contact(
+    v: hp.HPActionVariables,
+    contact: nycdb.models.Contact
+) -> None:
+    v.landlord_address_city_te = contact.address.city
+    v.landlord_address_street_te = contact.address.street_name
+    v.landlord_address_zip_te = contact.address.zipcode
+    v.landlord_address_state_mc = nycdb_addr_to_hp_state(contact.address)
+    v.service_address_full_te = ", ".join(contact.address.lines_for_mailing)
+    if isinstance(contact, nycdb.models.Company):
+        v.landlord_entity_or_individual_mc = hp.LandlordEntityOrIndividualMC.COMPANY
+    else:
+        v.landlord_entity_or_individual_mc = hp.LandlordEntityOrIndividualMC.INDIVIDUAL
+
+        # I don't think these are actually used in rendering the form, but just in case...
+        v.landlord_name_first_te = contact.first_name
+        v.landlord_name_last_te = contact.last_name
+    v.landlord_entity_name_te = contact.name
+    v.served_person_te = contact.name
+
+
+def fill_landlord_management_info_from_company(
+    v: hp.HPActionVariables,
+    mgmtco: nycdb.models.Company
+) -> None:
+    v.management_company_address_city_te = mgmtco.address.city
+    v.management_company_address_street_te = mgmtco.address.street_name
+    v.management_company_address_zip_te = mgmtco.address.zipcode
+    v.management_company_address_state_mc = nycdb_addr_to_hp_state(mgmtco.address)
+    v.service_address_full_management_company_te = ", ".join(
+        mgmtco.address.lines_for_mailing)
+    v.served_person_management_company_te = mgmtco.name
+    v.management_company_name_te = mgmtco.name
+
+    # TODO: We might actually want to fill this out even if we don't find
+    # a management company, as this could at least generate the required
+    # forms. Need to find this out.
```

Comment:
@romeboards I'm curious what you think of this TODO. Do we still want to fill out the fields asking if the management company should be sued, even if we don't find a management company for the user, so that hopefully the forms are in the packet just in case they decide to go that route?  Or alternatively, if the user has an individual owner landlord (not a company) does that mean there won't be a management company involved?